### PR TITLE
updated justinrainbow/json-schema

### PR DESCRIPTION
**Pull request by robinlehrmann**
Hi @theDisco can we update "justinrainbow/json-schema" to the new major Version ?
I use your extension with php7.4 and "justinrainbow/json-schema" in major version 4 is not compatible with it.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "php": ">=5.4.0",
         "behat/behat": "~3.3.0",
         "guzzlehttp/guzzle": "~6.0",
-        "justinrainbow/json-schema": "~4.1",
+        "justinrainbow/json-schema": "~5.1",
         "seromenho/xml-validator": "~v1.0"
```
Comment (jschaedl) [inline composer.json]: I think you should allow both: ~4.1I~5.0

File: composer.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         "php": ">=5.4.0",
         "behat/behat": "~3.3.0",
         "guzzlehttp/guzzle": "~6.0",
-        "justinrainbow/json-schema": "~4.1",
+        "justinrainbow/json-schema": "~5.1",
```
Comment (jschaedl) [inline composer.json]:
```suggestion
        "justinrainbow/json-schema": "~4.1I~5.0",
```

Comment (jschaedl) [inline composer.json]:
```suggestion
        "justinrainbow/json-schema": "~4.1|~5.0",
```

**Comment by theDisco** [comment]
Hey, sure. I will update travis setup to include newer versions of php and remove the older ones.

**Pull request merged**